route53_crossfeed_app.tf

```
@@ -295,6 +295,18 @@ resource "aws_route53_record" "crossfeed_staging_TXT" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "crossfeed_staging_ACME_TXT" {
```

I think this is more in line with the resource naming scheme we have used in this file:
```suggestion
resource "aws_route53_record" "crossfeed_staging_acme_TXT" {
```